robot.py

```diff
@@ -40,7 +50,7 @@ def autonomousInit(self) -> None:
         This method runs the autonomous command selected by your
         RobotContainer class.
         """
-        self.autonomousCommand = self.container.getAutonomousCommand()
+        self.autonomousCommand = self.container.get_autonomous_command()
```

Comment:
It looks to me like a `RobotContainer` has a method called `getAutonomousCommand()`, but no method called `get_autonomous_command()`.  I would expect that this won't work.  Does it work?